src/block.rs

```diff
@@ -146,6 +153,14 @@ pub struct BlockCore {
     creator: PublicKey,
     /// Total block reward being released in the block
     coinbase: u64,
+    /// Amount of SAITO left in reserve on the network
+    treasury: u64,
+    /// Start value in the `Burnfee` algorithm
+    start_burnfee: f64,
```

Comment:
Sorry if we keep repeating conversations, but can't I get the start_burnfee from previous block's burnfee?

Reply:
Nope, it's the opposite! You get the current burnfee form the previous block's `start_burnfee`